ManatiCR/fdi-front
# Update packages

## Steps to test
- `npm install`
- Edit `src/config.js` and change the `development` `apiUrl` to `http://a11y-fdi.pantheonsite.io/graphql`
- `npm run serve`
- Make sure all elements and sections of the app work as expected
